stylus/ui-app/alerts.styl

```diff
@@ -19,7 +19,15 @@ $alerts
         left       0
         background white
         opacity    .6
-        transition opacity ease-out .3s
+        visibility visible
+        transition opacity .3s, visibility 0s ease-out
+
+        &.coz-overlay--hidden
+            opacity          0
+            visibility       hidden
+            transition       opacity .3s, visibility 0s ease-in
+            transition-delay 0s, .3s
```

Comment:
Ok I think I got it 😆 
First timed value is duration and then you set the delay.
You can merge them by writing `transition: name duration delay, name duration delay`